src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { Comments } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+
```

Comment:
remove extra line
```suggestion
```

Reply:
I don't remove extra lines between imports because it's easier for me to navigate the code.